pkg/scheduler/frameworkext/eventhandlers/reservation_handler.go

```diff
@@ -122,10 +122,11 @@ func makeReservationErrorFunc(sched frameworkext.Scheduler, reservationLister sc
 func updateReservationStatus(client koordclientset.Interface, reservationLister schedulingv1alpha1lister.ReservationLister, rName string, schedulingErr error) {
 	err := util.RetryOnConflictOrTooManyRequests(func() error {
 		r, err := reservationLister.Get(rName)
-		if errors.IsNotFound(err) {
-			klog.V(4).Infof("skip the UpdateStatus for reservation %q since the object is not found", rName)
-			return nil
-		} else if err != nil {
+		if err != nil {
```

Comment:
This is about scheduler modifications, but the PR is to enhance descheduler. Please rollback irrelevant changes.

Reply:
done